lib/evss/auth_headers.rb

```diff
@@ -36,9 +37,15 @@ def eauth_json
           id: @user.ssn,
           edi: @user.edipi,
           firstName: @user.first_name,
-          lastName: @user.last_name
+          lastName: @user.last_name,
+          birthDate: iso8601_birth_date
         }
       }.to_json
     end
+
+    def iso8601_birth_date
+      return nil unless @user&.va_profile&.birth_date
+      DateTime.parse(@user.va_profile.birth_date).iso8601
```

Comment:
Will the birth date ever be off by 1 day since `DateTime.parse` takes timezone into account?  This looks similar to the fiasco we had on the front-end.

Reply:
👍  va_profile.birth_date (MVI) is sanely using only date, it's EVSS who require a datetime format:
```
> user.va_profile.birth_date
=> "19500423
> DateTime.parse(user.va_profile.birth_date)
=> Sun, 23 Apr 1950 00:00:00 +0000
> DateTime.parse(user.va_profile.birth_date).iso8601
=> "1950-04-23T00:00:00+00:00"
```

Reply:
@kreek  @omgitsbillryan Is the Post 9/11 GI Bill status date of birth currently parsed like this as well? Thinking we should be consistent.

Reply:
@raquelromano-gov EVSS services require "auth headers" (a.k.a. VAFI) with every request.  This is the piece where we create said headers.  This code is shared across any piece of `vets-api` code that calls an EVSS service.  So by way of code sharing, I suppose it's _already_ consistent.

Reply:
makes sense--thanks @omgitsbillryan !